Modules/zipimport.c

```diff
@@ -379,21 +405,29 @@ find_loader(ZipImporter *self, PyObject *fullname, PyObject **namespace_portion)
     return FL_MODULE_FOUND;
 }
 
+/*[clinic input]
+zipimport.zipimporter.find_module
+
+    fullname: unicode
```

Comment:
Do you prefer to keep Argument Clinic conversion separate from behavioral changes? Then we can add trailing `/`s to the argument lists and remove them in a separate PR.